arch/arc/core/mpu/arc_mpu_v3_internal.h

```diff
@@ -711,7 +711,7 @@ static int arc_mpu_init(struct device *arg)
 
 		/* record the static region which can be split */
 		if (mpu_config.mpu_regions[i].attr & REGION_DYNAMIC) {
-			if (dynamic_regions_num >
+			if (dynamic_regions_num >=
 			MPU_DYNAMIC_REGION_AREAS_NUM) {
 				LOG_ERR("no enough dynamic regions %d",
```

Comment:
now that you are here, you can change "no" to "not", as well.

Reply:
fair enough